ilp-core/src/main/java/org/interledger/core/Wrapped.java

```diff
@@ -0,0 +1,17 @@
+package org.interledger.core;
+
+import org.immutables.value.Value;
+import org.immutables.value.Value.Style.ImplementationVisibility;
+
+@Value.Style(
+  // Detect names starting with underscore
+  typeAbstract = "_*",
+  // Generate without any suffix, just raw detected name
+  typeImmutable = "*",
+  // Make generated public, leave underscored as package private
+  visibility = ImplementationVisibility.PUBLIC,
+  // Seems unnecessary to have builder or superfluous copy method
+  defaults = @Value.Immutable(builder = false, copy = false))
+public @interface Wrapped {
```

Comment:
What is this? I don't see if used anywhere.

Reply:
Oh never mind, guessing it gets used in another PR. Some Javadoc on this interface could be useful though.

Reply:
Yeah, I'll add some. This is used by Immutables to generate typed identifiers.

Reply:
This is fixed here: https://github.com/hyperledger/quilt/pull/244/commits/f1efe05b34f6fdffdd5c1494375f92a294fb8bc1